DecipherNow/gm-fabric-dashboard
# chore: automate versioning and changelog generation

Resolves #1539
## Proposed Changes

  - Automate the versioning process into a single command using npm version hooks
  
## How to Test

- Run the following:
```sh
git checkout automate-versioning
git checkout -b automate-versioning-test
npm version 1.0.6
```
  - Ensure that the command runs our linters and tests, bumps the version, generates the changelog, and updates the snapshots.
  - Ensure the documentation at https://docs.google.com/document/d/1JBHxBU-hN0Zbw9D1k9sNZfIIHPr9lVvY-uaWLhntztM reflects this new process
